pyproject.toml

```diff
@@ -39,11 +42,11 @@ packages = [
 
 [tool.poetry.dependencies]
 djangorestframework = "*"
-inflection = "^0.5.1"
+inflection = "^0.4"
```

Comment:
again, why the downgrade?

Reply:
It's not a downgrade, it's increasing the span we allow third parties to operate in. We're allowing
a user to use inflection in the range of `0.4 to  < 1.0` rather than `0.5.1 to < 1.0`. 

If we kept 0.5.1 as the lower bound and someone wanted to use this package, but were depending on `inflection==0.5.0`, Poetry would start raising resolver errors for them, when it's not strictly necessary.